openjdk.test.modularity/src/com.stf/net/adoptopenjdk/test/modularity/JlinkPluginOptionsTest.java

```diff
@@ -0,0 +1,379 @@
+/*******************************************************************************
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+package net.adoptopenjdk.test.modularity;
+
+import net.adoptopenjdk.stf.environment.DirectoryRef;
+import net.adoptopenjdk.stf.environment.ModuleRef;
+import net.adoptopenjdk.stf.extensions.core.StfCoreExtension;
+import net.adoptopenjdk.stf.plugin.interfaces.StfPluginInterface;
+import net.adoptopenjdk.stf.environment.StfTestArguments;
+import net.adoptopenjdk.stf.processes.definitions.JavaProcessDefinition.JarId;
+import net.adoptopenjdk.stf.runner.modes.HelpTextGenerator;
+import net.adoptopenjdk.stf.StfException;
+import net.adoptopenjdk.stf.util.StfDuration;
+import static net.adoptopenjdk.stf.extensions.core.StfCoreExtension.Echo.ECHO_ON;
+import net.adoptopenjdk.stf.environment.FileRef;
+import net.adoptopenjdk.stf.processes.ExpectedOutcome;
+import net.adoptopenjdk.stf.processes.definitions.JavaProcessDefinition;
+import net.adoptopenjdk.stf.processes.StfProcess;
+import net.adoptopenjdk.stf.processes.definitions.JmodDefinition;
+import net.adoptopenjdk.stf.processes.definitions.JlinkDefinition;
+
+import java.util.Properties;
+
+import java.io.File;
+import java.io.*;
+import org.junit.*;
+import org.junit.runner.*;
+
+/**
+ * 
+ * User story 4) 
+ * ~~~~~~~~~~~~~
+ * As a Java Product developer I can create a custom java runtime image using various 
+ * jlink plugin options. There are, in total, 13 plugin options available with the latest jdk release.
```

Comment:
Are there still 13 plugin options?

Reply:
It seems to have changed. I have removed the unnecessary line altogether from the comment. Thanks for pointing out!